src/auth/components/PasswordForm/style.less

```diff
@@ -28,19 +28,21 @@
 
     .input-box {
       border: none;
-      border-bottom: 1px solid @black;
+      border-bottom: 1px solid var(--theme-text-on-background-1);
       font-size: 16px;
       height: 28px;
       margin-bottom: 20px;
       padding-left: 34px;
       width: 300px;
+      background-color: var(--theme-background);
+      color: var(--theme-text-on-background-1);
```

Comment:
is this needed? (see above comment)

Reply:
Title required not required, input colors definitions required to match background in dark mode